app/views/content-page.html

```diff
@@ -1,7 +1,7 @@
 {% extends 'layouts/layout-nhsuk.html' %}
```

Comment:
There seems to be 2 `content-page` with different content

One at `/templates/content-page` (docs/views/templates/content-page.html) which is linked to from the page templates and one at `/templates/content-page` (app/views/content-page), which as far as I can see isn't linked to but has more content. They used to have the same content but have diverged, which is the one to keep?

Reply:
We need both, I have renamed the one in app to page-example.html to save confusion.